_includes/specials/taproot/en/12-backups.md

```diff
@@ -0,0 +1,101 @@
+In [last week's column][FIXME], Antoine Poinsot described how taproot
+can make [vault][topic vaults]-style coin backup and security schemes
+more private and fee efficient.  In this week's column, we'll look
+several other backup and security schemes that are improved by
+converting to taproot.
+
+- **Simple 2-of-3:** as [mentioned][threshold signing] in a previous
+  week, it's easy to use a combination of [multisignatures][topic
+  multisignature] and scriptpath spends to create a 2-of-3 spending
+  policy that's normally just as efficient onchain as a single-sig spend
+  and which is much more private than current P2SH and P2WSH multisigs.
```

Comment:
perhaps "more private and cheaper than" (since the happy case keypath spend is a single schnorr signature)

Reply:
I think this is covered by the previous clause saying that it's "just as efficient onchain as a single-sig spend".  That's more specific than just saying it's cheaper than P2WSH, which may be the case even if multisig P2WSH is converted to tapscript due to tapscript's smaller signature sizes.

Reply:
ok. That requires the reader to know that a single-sig spend is cheaper than a P2SH or P2WSH multisig spend, but hopefully our readers know that already!